Source/common/SNTConfigurator.m

```diff
@@ -1038,6 +1045,18 @@ - (BOOL)enableForkAndExitLogging {
   return number ? [number boolValue] : NO;
 }
 
+- (NSArray<NSString *> *)telemetry {
+  NSArray *events = self.configState[kTelemetryKey];
+
+  for (id event in events) {
+    if (![event isKindOfClass:[NSString class]]) {
+      return nil;
+    }
+  }
```

Comment:
I changed this up a bit after thinking about it... If someone attempts to filter, "failing open" and allowing all events seems wrong. Instead, it will now strip out non-string things in a sort of "fail closed" operation. Added a comment to explain.